src/controlCenter/controlCenter.vala

```diff
@@ -207,6 +210,22 @@ namespace SwayNotificationCenter {
             });
         }
 
+        public void on_realize () {
```

Comment:
Maybe add a short comment above like:
```vala
/** Sets the Window Width and Height and clamps the values to the monitors size */
```

Also this method needs to be called in `on_visibility_change`